src/Pact/Gas/Table.hs

```diff
@@ -183,12 +183,21 @@ defaultGasTable =
 
   ]
 
+{-# NOINLINE defaultGasTable #-}
```

Comment:
Can this be by `defaultGasTable`'s function definition?

Reply:
It's right after it ... ?

Reply:
ah yes, sorry! I misread it for `defaultGasModel`